scripts/server.js

```diff
@@ -19,7 +19,7 @@ const nameify = (str) =>
 const srces = {
   css: 'src/plugin.scss',
   js: 'src/plugin.js',
-  tests: glob.sync('test/**/*.test.js')
+  tests: glob.sync('test/**/test.*.js')
```

Comment:
This is to have shorter, easier-to-read file names that still all start with the same so they sort together